example/no_tls_both.cpp

```diff
@@ -279,9 +279,7 @@ void server_proc(Server& s, std::set<con_sp_t>& connections, mi_sub_con& subs) {
                         r.first->con->publish(
                             boost::asio::buffer(topic_name),
                             boost::asio::buffer(contents),
-                            std::min(r.first->qos_value, pubopts.get_qos()) | pubopts.get_retain(),
-                            MQTT_NS::v5::properties{},
-                            std::make_pair(topic_name, contents)
+                            std::min(r.first->qos_value, pubopts.get_qos())
```

Comment:
You are removing the std::make_pair() life-keeper?

Reply:
Yes because topic_name and contents has own buffer internally. It is key feature for receive callback.

Reply:
This example is passing topic_name and contents as boost::asio::buffer, which discards life-keeping functionality.

Either put the std::make_pair() life-keeping back, or remove the call to "boost::asio::buffer()"

Reply:
You are right. I will remove o "boost::asio::buffer()".